pkg/pmem-csi-operator/controller/add_deployment.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package controller
+
+import (
+	"github.com/intel/pmem-csi/pkg/pmem-csi-operator/controller/certificates"
+	"github.com/intel/pmem-csi/pkg/pmem-csi-operator/controller/deployment"
+)
+
+func init() {
+	// AddToManagerFuncs is a list of functions to create controllers and add them to a manager.
+	AddToManagerFuncs = append(AddToManagerFuncs, certificates.Add)
+	AddToManagerFuncs = append(AddToManagerFuncs, deployment.Add)
```

Comment:
Nit: this can be a single `append()`.

Reply:
> Nit: this can be a single `append()`.

With the [new change](https://github.com/intel/pmem-csi/pull/525/commits/79e3841e8b68b0c8194257ed9d3866f6593327a2#diff-a6eb5181b8c90da9284577844583a624), this Nit is not valid anymore ;-)